apps/dot_ext/tests/test_api_error_codes.py

```diff
@@ -0,0 +1,184 @@
+from apps.test import BaseApiTest
+from http import HTTPStatus
+import json
+import pytz
+from random import randint
+from apps.authorization.models import (
+    DataAccessGrant,
+)
+from apps.constants import AccessType
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from unittest import mock
+
+
+class StubDate(datetime):
+    pass
+
+
+# These tests cover `validate_app_is_active` in `utils.py`
+# In particular, they cover whether or not we get the right error
+# codes back for a given call/situation.
+class TestDataAccessPermissions(BaseApiTest):
+
+    # In this test, we should get back a `HTTPStatus.UNAUTHORIZED` when we try
```

Comment:
Another case where it's probably for the best to just remove the comment, but if we want to keep the comment, we need it to say FORBIDDEN.

Reply:
I removed the (response_code=XYZ) comments in general. They're redundant with the code, and the code is clear because of how it is constructed.